api/Dockerfile

```diff
@@ -29,17 +29,17 @@ FROM ${TARGET_BASE_IMAGE}
 RUN \
     set -x \
     && apt-get update \
-    && apt-get install -y openssl libssl-dev ca-certificates curl jq  android-tools-adb \
-    && curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
-    && chmod +x get_helm.sh \
-    && ./get_helm.sh \
+    && apt-get install -y openssl libssl-dev ca-certificates curl jq wget android-tools-adb \
+    && wget https://get.helm.sh/helm-v3.14.0-linux-arm64.tar.gz \
```

Comment:
We hardcoded the arm64 version in dockerfile, but I saw there're different helm packages for different platform. Should we detect the target platform and find the correct URL? 
<img width="715" alt="image" src="https://github.com/user-attachments/assets/a9480313-a28f-41dc-8a75-c3e499b16d54">